google/gax/auth.py

```diff
@@ -31,19 +31,49 @@
 
 from __future__ import absolute_import
 
+import grpc
```

Comment:
Sorry, posted that comment before I saw yours. Moving it to a private module SGTM.